examples/raphtory-example-lotr/src/main/scala/com/raphtory/examples/lotrTopic/FileOutputRunner.scala

```diff
@@ -34,4 +35,13 @@ object FileOutputRunner extends App {
     .writeTo(output)
 
   queryHandler.waitForJob()
+
+  val queryHandlerCC = graph
+    .at(32674)
+    .past()
+    .execute(ConnectedComponents())
```

Comment:
This need to stay as Degree Separation but I can change back in my tutorial check through